src/rastervision/tagging/tasks/predict.py

```diff
@@ -87,7 +88,27 @@ def compute_probs(run_path, model, options, generator, split):
         normalize=True, only_xy=False)
 
     for batch_ind, batch in enumerate(split_gen):
-        y_probs.append(model.predict(batch.x))
+        # Performing safe augmentations on images one by one, predicting
+        # probs and taking average, if calculating test probabilities
+        if split == TEST:
+            for img_ind in range(batch.x.shape[0]):
+                img = batch.x[img_ind, :, :, :]
+                rot_batch = []
+                for rotation in range(4):
+                    rot_img = np.rot90(img, rotation)
+                    rot_img = np.expand_dims(rot_img, axis=0)
+                    rot_batch.append(rot_img)
+                rot_batch.append(np.flipud(rot_img))
+                rot_batch.append(np.fliplr(rot_img))
+                rot_batch = np.concatenate(rot_batch, axis=0)
+
+                rot_probs = np.mean(model.predict(rot_batch), axis=0)
+                rot_probs = np.expand_dims(rot_probs, axis=0)
```

Comment:
If you do `rot_probs = np.mean(model.predict(rot_batch), axis=0, keepdims=True)`, then you can get rid of this line.